client/src/components/Todo-List/TodoList.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from "react";
+import Todo from "../Todo/Todo";
+import "../Todo-List/TodoList.scss";
+
+const TodoList = () => {
+	let [value, setValue] = useState("");
+	let [list, setList] = useState([]);
+	useEffect(() => {
+		getItemsFromServer();
+	}, []);
+
+	const getItemsFromServer = async () => {
+		await getItems().then((todos) => {
+			setList(todos);
+		});
+	};
+
+	const getItems = async () => {
+		const response = await fetch("http://localhost:3000/items");
+		const todos = await response.json();
+		return todos;
+	};
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+	};
+
+	const handleTrashClicked = (id) => {
+		deleteItem({ id }).then(() => {
+			getItemsFromServer();
+		});
+	};
+
+	const handleCheckClicked = (id, statusItem) => {
+		const isDone = ({ statusItem } = true);
+		postIsCompleted(id, isDone).then(() => {
+			getItemsFromServer();
+		});
+	};
+
+	const postIsCompleted = async (id, isDone) => {
+		await fetch(`http://localhost:3000/item/${id}`, {
+			method: "POST",
+			headers: { "Content-Type": "application/json" },
+			body: JSON.stringify({ id, isDone }),
+		});
+	};
+
+	const handleClick = async (e) => {
+		const response = await postItem(value);
+		console.log(response);
+		const newItem = await getItemsFromServer();
+		setValue("");
+	};
```

Comment:
Don't create blocking functions with async await like here - This can cause a lot of bugs and inconsistency.
The only async await should happen inside a useEffect in react 

Doing this will make the UI stuck until both postItem and getItemsFromServer are resolved, will set the state value with a different value which will then re-render the component anyway triggering you useeEffect which does getItemsFromServer again